spartan/scripts/test_kind.sh

```diff
@@ -109,7 +109,14 @@ ethereum_slot_duration=$(./read_value.sh "ethereum.blockTime" $value_yamls)
 aztec_slot_duration=$(./read_value.sh "aztec.slotDuration" $value_yamls)
 aztec_epoch_duration=$(./read_value.sh "aztec.epochDuration" $value_yamls)
 aztec_proof_submission_window=$(./read_value.sh "aztec.proofSubmissionWindow" $value_yamls)
-l1_account_mnemonic=$(./read_value.sh "aztec.l1DeploymentMnemonic" $value_yamls)
+
+if [ "$sepolia_run" = "true" ]; then
+  # Read the mnemonic from file mnemonic.tmp
+  l1_account_mnemonic=$(cat mnemonic.tmp)
```

Comment:
Heads up there's a `-x` above in the file that may leak this. I'd set a `+x` around here to make sure it's not shown in logs.